build.gradle

```
@@ -32,7 +32,20 @@ test {
     }
 }
 
-if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre17")){
+if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre18")){
```

nit: spacing for brace

Just like line 61 [here](https://github.com/microsoft/mssql-jdbc/pull/1802/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R61)

```suggestion
if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre18")) {
```